.github/workflows/release.yml

```diff
@@ -13,51 +13,69 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'release version in format 7.y.z'
+        description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
+        default: '7.y.z'
```

Comment:
I'd suggest having an empty default value, so we wouldn't have accidental workflow runs with `7.y.z` taken as a default parameter.